services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -169,6 +170,7 @@ async def get_dynamic_sidecar_status(
     summary="stops previously spawned dynamic-sidecar",
 )
 async def stop_dynamic_service(
+    request: Request,
```

Comment:
Use dependency injection instead of directly ``request`` to get settings. 

Please avoid using request directly in the handlers. Using dependency injection leaves the interface with "what it needs" and allows high reusability and testability.

Reply:
Good point. Also took the opportunity to remove some more cases where dependency injection could be used.